pkgs/data/fonts/commit-mono/default.nix

```
@@ -0,0 +1,35 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchzip,
+}:
```

```suggestion
{ lib
, stdenvNoCC
, fetchzip
}:
```
Almost missed it but currently our formatting for inputs is like my suggestion above :/ this will probably change with the formatting rfc.

---

Committing this as well